### PR TITLE
Some syntax modifications to the analytics dimensions

### PR DESCRIPTION
**Pull request by kenzieschmoll**
From the gtags documentation, it appears the dimensions should be in quotes: https://developers.google.com/analytics/devguides/collection/gtagjs/custom-dims-mets

### CODE REVIEW

File: packages/devtools_app/web/devtools_analytics.js [hunk 1]
```diff
@@ -44,18 +44,18 @@ function initializeGA() {
...
         // Custom dimensions:
-        dimension1: 'user_app',
-        dimension2: 'user_build',
-        dimension3: 'user_platform',
-        dimension4: 'devtools_platform',
-        dimension5: 'devtools_chrome',
-        dimension6: 'devtools_version',
-        dimension7: 'ide_launched',
-        dimension8: 'flutter_client_id',
+        'dimension1': 'user_app',
```
Comment (jacob314) [inline packages/devtools_app/web/devtools_analytics.js:47]: in JS, writing dimension1 in this position is equivalent to writing 'dimension1' so this shouldn't change any actual functionality but I agree writing 'dimension1' is clearer.

File: packages/devtools_app/lib/src/analytics/analytics.dart [hunk 2]
```diff
@@ -475,12 +475,12 @@ void setupAndGaScreen(String screenName) async {
   }
 }
 
-void setupDimensions() {
+Future<void> setupDimensions() async {
```
Comment (jacob314) [inline packages/devtools_app/lib/src/analytics/analytics.dart:478]: should we now be awaiting this function everywhere we are calling it?

Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/analytics/analytics.dart:478]: the only place we call this is in `DevToolsAppState.initState`, so we cannot await it there. Regardless of this though, I think it's better to have the proper return type here so we can await `computeFlutterClientId` before setting the `_analyticsComputed` bit to true at the end of this method.

**Comment by jacob314** [comment]
lgtm

**Pull request merged**